client/src/lib.rs

```diff
@@ -494,15 +452,18 @@ impl<V: vdaf::Client<16>> ClientBuilder<V> {
 }
 
 /// A DAP client.
+pub type Client<V> = ClientWithClock<RealClock, V>;
```

Comment:
Since this is on `release/0.7`, I'm trying to avoid breaking the exported API. Other crates in the workspace that pull in `janus_client` still build.